contracts/container/contract.go

```diff
@@ -491,6 +493,100 @@ func List(owner []byte) [][]byte {
 	return list
 }
 
+// AddNextEpochNodes accumulates passed nodes as container members for the next
+// epoch to be committed using [CommitContainerListUpdate]. Results of the call
+// operation can be received via [Nodes]. This method must be called only when
+// a container list is changed, otherwise nothing should be done.
+// Call must be signed by the Alphabet nodes.
+func AddNextEpochNodes(cID interop.Hash256, publicKeys []interop.PublicKey) {
+	if len(cID) != interop.Hash256Len {
+		panic(cst.ErrorInvalidContainerID + ": length: " + std.Itoa10(len(cID)))
+	}
+
+	ctx := storage.GetContext()
+	multiaddr := common.AlphabetAddress()
+	common.CheckAlphabetWitness(multiaddr)
+
+	counter := 0
+	counterKey := append([]byte{nextEpochNodesPrefix, 'i'}, cID...)
```

Comment:
> isnt this overcomplicated?

maybe. i did not want to have even more prefixes for the same feature, this is my suggestion. can add a dedicated counter prefix if @roman-khimov is also for it

> Collission, although unlikely, is possible

to have problems we need either 31 of 32 bytes should match (with an offset) and the first one should be fixed `i` byte OR a container should have the first 31 bytes of `i`. it is very like "two different containers have the same ID" with almost the same number of problems for NeoFS

Reply:
You can avoid storing the counter if you're to take the last element of `nextEpochNodesPrefix + cID` which is easy with `Backwards` flag (https://pkg.go.dev/github.com/nspcc-dev/neo-go/pkg/interop@v0.0.0-20241007161539-0968c3a81fb0/storage#FindFlags). But to do this you need to have `counter` serialized in BE form.

Reply:
Or you can avoid using counter with https://pkg.go.dev/github.com/nspcc-dev/neo-go/pkg/interop@v0.0.0-20241007161539-0968c3a81fb0/contract#CreateStandardAccount. This would solve duplicate problem at the same time.

Reply:
@roman-khimov, idea with `Backwards` is the best for me, i like it

> But to do this you need to have counter serialized in BE form.

how can i ensure it? are there any contract side helpers for it?